examples/futures/testnet.ts

```diff
@@ -0,0 +1,54 @@
+import { RestClient } from '../../src/index.js'; // For an easy demonstration, import from the src dir. Normally though, see below to import from the npm installed version instead.
+// import { RestClient } from 'gateio-api'; // Import the RestClient from the published version of this SDK, installed via NPM (npm install gateio-api)
+
+// Define the account object with API key and secret
+const account = {
+  key: process.env.API_KEY || 'yourApiHere', // Replace 'yourApiHere' with your actual API key
+  secret: process.env.API_SECRET || 'yourSecretHere', // Replace 'yourSecretHere' with your actual API secret
+};
+
+// Initialize the RestClient with the API credentials
+const gateRestClient = new RestClient({
+  apiKey: account.key,
+  apiSecret: account.secret,
+  /**
+   * To use a different base URL, use the baseUrl key. The SDK uses the live environment by default:
+   * baseUrlKey: 'live',
+   *'https://api.gateio.ws/api/v4'
+   * But you can force it to use any of the available environments. Examples below.
+   */
```

Comment:
```suggestion
  /**
   * To use a different base URL, use the baseUrl key. The SDK uses the live environment by default:
   * baseUrlKey: 'live',
   *
   * But you can force it to use any of the available environments. Examples below.
   */
```

I'd remove the URL itself to avoid confusion